test/e2e/setup/aro/README.md

```diff
@@ -0,0 +1,91 @@
+# Maestro ARO-HCP Env Setup
+
+## Prerequisites
+
+* `az` version >= 2.60, `jq`, `make`, [kubelogin](https://azure.github.io/kubelogin/install.html), `kubectl` version >= 1.30, `helm`
+* `az login` with service principal (azure AD user support is WIP)
+
+### Create Service Cluster
+
+Change those flags accordingly and then run the following command. Depending on the selected features, this may take a while:
+
+  ```bash
+  AKSCONFIG=svc-cluster make cluster
+  ```
+
+### Create Management Cluster
+
+A Management Cluster depends on certain resources found in the resource group of the Service Cluster. Therefore, a standalone Management Cluster can't be created right now and requires a Service Cluster
+
+  ```bash
+  AKSCONFIG=mgmt-cluster make cluster
+  ```
+
+### Access AKS Clusters
+
+   ```bash
+   AKSCONFIG=svc-cluster make aks.admin-access  # one time
+   AKSCONFIG=svc-cluster make aks.kubeconfig
+   AKSCONFIG=svc-cluster export KUBECONFIG=${HOME}/.kube/${AKSCONFIG}.kubeconfig
+   kubectl get ns
+   ```
+
+    (Replace `svc` with `mgmt` for management clusters)
+
+### Cleanup
+
+Setting the correct `AKSCONFIG`, this will cleanup all resources created in Azure
+
+   ```bash
+   AKSCONFIG=svc-cluster make clean
+   ```
+
+    (Replace `svc` with `mgmt` for management clusters)
+
+## Deploy Maestro to AKS Clusters
+
+### Maestro Server
+
+> Make sure your `KUBECONFIG` points to the service cluster!!!
+
+> The service cluster has no ingress. To interact with the services you deploy use `kubectl port-forward`
```

Comment:
To interact with the services you need to use `kubectl port-forward`

Reply:
done.